docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -86,8 +84,6 @@ This limit will be applied to every container of the given component.
 +
 For the `plugin` components, CPU limits can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-If none of them are specified, system-wide defaults are applied (see description of `pass:[CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__LIMIT__CORES]` system property).
```

Comment:
Based on Mario's previous comment, I think he mentioned that we need to add "From a devfile spec we should say that, if they are not specified, the values are undetermined: they may (or may not) be inferred from the application that consumes the devfile or from Kubernetes." after removing the existing line.

Reply:
Thanks, Elson, I overlooked that! I'll address these comments shortly.